editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -524,7 +524,12 @@ function navigatorStructure(editorState: EditorState, deriveState: DerivedState)
   const lines = deriveState.visibleNavigatorTargets.map((target) => {
     const targetAsText = navigatorEntryToKey(target)
     let prefix: string = ''
-    const depth = navigatorDepth(target, editorState.jsxMetadata)
+    const depth =
+      deriveState.navigatorRows.find((row) =>
+        isRegulaNavigatorRow(row)
```

Comment:
minor but you can use `getEntriesForRow` instead of the hand rolled ternary here

Reply:
https://github.com/concrete-utopia/utopia/pull/5948/commits/0272a1754a8231bef5b88ad9533105096013b09f